slices/grep.yaml

```diff
@@ -0,0 +1,20 @@
+package: grep
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre3_libs
+    contents:
+      /bin/grep:
+
+  deprecated:
+    # These are shell scripts requiring a symlink from /usr/bin/bash to /bin/sh
+    # See: https://manpages.ubuntu.com/manpages/jammy/en/man1/grep.1.html
+    essential:
+      - bash_bins
+      - libpcre3_libs
```

Comment:
I think we will need `grep_bins` since the `egrep` (and others) call `grep` from inside. 

```sh
~# cat grep/usr/bin/egrep 
#!/bin/sh
cmd=${0##*/}
exec grep -E "$@"
```

We can remove the `libpcre2-8-0` dependency from this slice since `grep_bins` will have it.